kube/kubetypes/types.go

```diff
@@ -6,6 +6,7 @@ package kubetypes
 const (
 	// Hostinfo App values for the Tailscale Kubernetes Operator components.
 	AppOperator          = "k8s-operator"
+	AppProxy             = "k8s-proxy"
```

Comment:
Could we add a comment as to what we mean by this app type? Presumably this is going to be the generic name of the `k8s-proxy` devices so that we will no longer be able to distinguish which tailnets use ingress/egress/api server proxy, which is a bit of a downside